### PR TITLE
refactor(FAB): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
supports #1654 #1500

This already had @apply for most, just added it for bg trasparent. 

Not sure if I need to do something with the @extend

### CODE REVIEW

File: src/components/calcite-fab/calcite-fab.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :host {
   @extend %component-host;
```
Comment (paulcpederson) [inline src/components/calcite-fab/calcite-fab.scss:2]: you could remove this and put in the individual properties of the class:

```
  box-sizing: border-box;
  * {
    box-sizing: border-box;
  }
  background-color: var(--calcite-ui-foreground-1);
  color: var(--calcite-ui-text-2);
  font-size: var(--calcite-font-size--1);
```

Honestly I'm not sure why some of these are on here as they don't do anything...

Comment (paulcpederson) [reply src/components/calcite-fab/calcite-fab.scss:2]: the spacing vars aren't even used here? I think you could remove that one.

Comment (julio8a) [reply src/components/calcite-fab/calcite-fab.scss:2]: I can try removing them and see if screener has any changes?

**Comment by paulcpederson** [comment]
Screener is rightly calling out a visual regression here.

**Comment by julio8a** [comment]
@paulcpederson, could this be a screener bug, when I updated the code and now, it looks fine on my local build:
image

**Comment by paulcpederson** [comment]
Are all your changed from local on this branch?

**Comment by julio8a** [comment]
Yes, can you test it by checking out the branch?

**Comment by paulcpederson** [comment]
@julio8a no, it's not fine in your local build, your branch also shows this problem. Change the background to make it easier to see:

Screen Shot 2021-03-05 at 9 18 11 AM

**Comment by julio8a** [comment]
Sorry about that. Okay, I fiiiiiixed it. Come on screener, help me out.

**Comment by julio8a** [comment]
The screener came through, I think it's good to merge, just need 1 approval.

**Comment by bstifle** [comment]
idk how to read screener what was the issue

**Comment by julio8a** [comment]
There are no visual changes so screener won't show anything. 

When screener check has a green check means it passed and there were no visual changes. @bstifle 

image

**Comment by julio8a** [comment]
Can someone approve?

**Review by caripizza — APPROVED** [review]

**Pull request merged**